solutions/swb-reference/README.md

```diff
@@ -6,348 +6,728 @@ The 0.1.0 beta release is available for customers to test the newly re-architect
 
 This release includes:
 
-* New user experience for workspace based on the AWS UI Design System
 * Sagemaker Notebook as a compute environment (connect, start, stop, terminate)
 * Data set attachment to Sagemaker Notebooks
 * Research user creation
 * Workspaces include search/filter/sort functions
 * Customers able to add custom environments
 * Improved performance on workspace page with pagination
 
-To test the new Service Workbench on AWS v2 r0.1.0, follow the deployment instructions:
-
-* Setup Instructions for [SWBv2p1](./SETUP_v2p1.md)
-* Adding a [new environment type](./AddingNewEnvironmentType.md)
-
-*Note:* API based updates to the database are required for some functionalities.
-
 # Code Coverage
 
 | Statements                  | Branches                | Functions                 | Lines             |
 | --------------------------- | ----------------------- | ------------------------- | ----------------- |
 | ![Statements](https://img.shields.io/badge/statements-95.42%25-brightgreen.svg?style=flat) | ![Branches](https://img.shields.io/badge/branches-84.5%25-yellow.svg?style=flat) | ![Functions](https://img.shields.io/badge/functions-88.63%25-yellow.svg?style=flat) | ![Lines](https://img.shields.io/badge/lines-95.37%25-brightgreen.svg?style=flat) |
 
-## Requirements
 
-The requirements below are for running the lambda locally
 
-1. Install SAM CLI ([link](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html))
-2. Install Docker ([link](https://docs.docker.com/get-docker/))
+# Setup Instructions for SWBv2p1
+
+Follow the instructions below to deploy SWBv2 with Sagemaker Notebook lifecycle support. You'll be able to 
+launch/start/stop/terminate/connect to a Sagemaker Notebook instance.
+
+## Prerequisite
+* An AWS account for deploying SWBv2 API. This account will be called the `Main Account`.
+   * On the command line, set your [credential file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html) to have your `Main Account` as the `default` profile 
+* An AWS account for hosting environments. This account will be called the `Hosting Account`.
+* Software
+  * [Rush](https://rushjs.io/pages/developer/new_developer/) v5.62.1 or later. We'll be using this tool to manage the packages in our mono-repo
+  * Node 14.x or 16.x [(compatible node versions)](https://github.com/aws-solutions/solution-spark-on-aws/blob/main/rush.json#L9)
+  * [POSTMAN](https://www.postman.com/) (Optional) This is used for making API requests to the server. POSTMAN is not needed if you already have a preferred API client. 
```

Comment:
not blocking: we may want to recommend a new API client

Reply:
IMO this should be handled by the task to change postman as it may need some considerable changes